code/modules/overmap/internet/internet_circuitboards.dm

```diff
@@ -34,3 +34,12 @@
 		/obj/item/stock_parts/keyboard = 1,
 		/obj/item/stock_parts/power/terminal = 1
 	)
+
+/datum/fabricator_recipe/imprinter/circuit/internet_uplink
+	path = /obj/item/stock_parts/circuitboard/internet_uplink
+
+/datum/fabricator_recipe/imprinter/circuit/internet_uplink_computer
+	path = /obj/item/stock_parts/circuitboard/internet_uplink_computer
```

Comment:
This is probably how the computer one should be built, but I think it _also_ fails to set the board_type to computer.

Reply:
That's true, it doesn't have the right board type. I'll fix that.

Reply:
Nah, the default value of the board_type is ``"computer"``. It should be fine.